lib/ansible/modules/cloud/azure/azure_rm_azurefirewall_info.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
```

Comment:
pls exclude firewall for now from this commit, this one is a bit more complex and will require some more work.
gallery modules are simpler and higher priority